README.md

```diff
@@ -29,7 +29,42 @@ On many systems, the following hugepages setup is sufficient:
 
 ### Option 1: Obtain Packages
 
-Debian packages for Gatekeeper aren't yet available, but will be soon.
+Debian packages for Gatekeeper aren't yet available, but will be soon at the
+[Releases](https://github.com/AltraMayor/gatekeeper/releases)
+page.
```

Comment:
Please write as it were already available, so we don't need to patch it once we release the RC1.

Reply:
Done.